packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -28,17 +28,34 @@ const initialValue = [
   },
 ];
 
+const mixedInitialValue = [
+  {
+    type: 'heading',
+    level: 1,
+    children: [{ type: 'text', text: 'A heading one' }],
+  },
+  {
+    type: 'paragraph',
+    children: [{ type: 'text', text: 'A line of text in a paragraph.' }],
+  },
+  {
+    type: 'heading',
+    level: 2,
+    children: [{ type: 'text', text: 'A heading two' }],
+  },
+];
+
 const user = userEvent.setup();
 
 const baseEditor = createEditor();
 
-const Wrapper = ({ children }) => {
+const Wrapper = ({ children, initialData }) => {
```

Comment:
At this point should we just inline this wrapper component? I'm not sure I understand the benefits of it being standalone :)

Reply:
If it is ok for you I am gonna stay with the Wrapper component because I need a component to define the `const [editor] = React.useState(() => withReact(baseEditor));` state inside it and with the inline definition it is not possible to use a state definition inside a function

Reply:
> not possible to use a state definition inside a function

How come? 🤔

Reply:
<img width="1088" alt="error" src="https://github.com/strapi/strapi/assets/2589748/0774b515-f1cc-400a-a5bd-e9b19589b4c1">